lib/foreman_maintain/cli/base.rb

```diff
@@ -163,11 +165,16 @@ def self.interactive_option
         option(['-w', '--whitelist'], 'whitelist',
                'Comma-separated list of labels of steps to be skipped') do |whitelist|
           raise ArgumentError, 'value not specified' if whitelist.nil? || whitelist.empty?
+
           whitelist.split(',').map(&:strip)
         end
 
         option ['-f', '--force'], :flag,
                'Force steps that would be skipped as they were already run'
+
+        option '--disable-spinner', :flag, 'Disable the spinner' do |disable_spinner|
```

Comment:
We might want to make it clear that this option also prints the output in plaintext.  Or perhaps the option could simply be `--plaintext` ?